### ISSUE TITLE
Get some Errors with example/rnn/lstm_ptb.py

### ISSUE THREAD

**Issue by to-shimo** [dmlc/mxnet#1322]
I'm trying to get start mxnet with an example/rnn/lstm_ptb.py. 
But i don't know what happened with it:
"
Loading ./data/ptb.train.txt, size of data = 929590
Unique token: 10000
Loading ./data/ptb.valid.txt, size of data = 73761
Unique token: 10000
Vocab=10000
Training with train.shape=(46445, 20)
Training with val.shape=(3675, 20)
batch_size=20
seq_len=35
/home/lgh/soft/mxnet/example/rnn/lstm.py:184: RuntimeWarning: divide by zero encountered in log
  nll = -np.sum(np.log(seq_label_probs.asnumpy())) / len(X[0,:])
Epoch [28] Train: NLL=inf, Perp=inf
"

thx.

p.s. 
my steps:
1. get_ptb_data.sh
2. python lstm_ptb.py

**Comment by jermainewang** [comment]
Have you changed learning rate? It seems that learning rate may be too large.

**Comment by to-shimo** [comment]
@jermainewang not yet

**Comment by antinucleon** [comment]
You may try to change rmsprop to sgd. It is break of recent change in optimizer and will be fixed soon.

**Comment by to-shimo** [comment]
@antinucleon thx, it works well.

**Comment by futurely** [comment]
The Google Inception v4 authors used RMSProp to train their best models. But it seems that training with RMSProp leads to no accuracy improvement. What parts of RMSProp are broken and when will them be fixed?
